documentation/book/con-product-downloads.adoc

```diff
@@ -3,14 +3,23 @@
 // getting-started.adoc
 
 [id='downloads-{context}']
-= {ProductName} downloads
+= Installing {ProductName} and deploying components
 
 ifdef::Downloading[]
-{ProductName} releases are available to download from {ReleaseDownload}. The release artefacts contain documentation, installation, and example `.yaml` files for deployment on {ProductPlatformName}. The installation, and example files are used throughout this documentation. Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
+To install {ProductName}, download the release artefacts from {ReleaseDownload}.
+
+The folder contains several YAML files to help you deploy the components of {ProductName} to {ProductPlatformName}, perform common operations, and configure your Kafka cluster. The YAML files are referenced throughout this documentation.
+
+Additionally, a Helm Chart is provided for deploying the Cluster Operator using link:https://helm.sh/[Helm^]. The container images are available through the {DockerRepository}.
```

Comment:
Do we really need two big blocks which are separate? I think that this is really the only paragraph which is upstream only. It should be possible to just use variables for the rest.

Reply:
I've modified the file to use variables for all content apart from anything that's common to both upstream and downstream.